internal/pkg/manifest/validate.go

```diff
@@ -85,6 +85,12 @@ func (l LoadBalancedWebService) Validate() error {
 // Validate returns nil if LoadBalancedWebServiceConfig is configured correctly.
 func (l LoadBalancedWebServiceConfig) Validate() error {
 	var err error
+	if l.RoutingRule.Disabled() && l.NLBConfig.IsEmpty() {
+		return errors.New(`must have one of "http" or "nlb" enabled`)
```

Comment:
maybe we can use `errFieldMutualExclusive` with `mustExist` on?

Reply:
I'm using `errAtLeastOneFieldMustBeSpecified`!